src/apidoc.rs

```diff
@@ -105,9 +80,10 @@ impl MosecOpenAPI {
     ///
     /// [path_item]: https://spec.openapis.org/oas/latest.html#path-item-object
     /// [utoipa-gen]: https://crates.io/crates/utoipa-gen
-    pub fn replace_path_item(&mut self, from: &str, to: &str) {
-        if let Some(path_item) = self.api.paths.paths.remove(from) {
-            self.api.paths.paths.insert(to.to_owned(), path_item);
+    pub fn replace_path_item(&mut self, from: &str, to: &str) -> &mut Self {
+        if let Some(r) = self.api.paths.paths.remove(from) {
+            self.api.paths.paths.insert(to.to_owned(), r);
         }
+        self
```

Comment:
Never been used?

Reply:
To allow method chaining

Reply:
If there's more than one route to be replaced, it would look cleaner